routes/shoutouts.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const service = require('../services/shoutouts');
+
+const router = express.Router();
+
+router.get('/:userId', service.getShoutouts);
```

Comment:
Let the routes layer handle the request object and return anything related to the HTTP transport layer like a status code or headers from the service layer.